pkg/virt-controller/watch/migration_test.go

```diff
@@ -305,16 +305,15 @@ var _ = Describe("Migration watcher", func() {
 		if len(vmi.Labels) == 0 {
 			vmi.Labels = nil
 		}
-		controller.vmiStore.Add(vmi)
-		key, err := virtcontroller.KeyFunc(vmi)
+		err := controller.vmiStore.Add(vmi)
 		Expect(err).To(Not(HaveOccurred()))
-		mockQueue.Add(key)
 		_, err = virtClientset.KubevirtV1().VirtualMachineInstances(vmi.Namespace).Create(context.Background(), vmi, metav1.CreateOptions{})
 		Expect(err).ToNot(HaveOccurred())
 	}
 
 	addMigration := func(migration *virtv1.VirtualMachineInstanceMigration) {
-		controller.migrationIndexer.Add(migration)
+		err := controller.migrationIndexer.Add(migration)
+		Expect(err).To(Not(HaveOccurred()))
```

Comment:
nit: 
```go
Expect(controller.migrationIndexer.Add(migration)).To(Succeed())
```

Reply:
What I wrote matches what the other functions do.
We could change them all, but it might be a bit out of scope for this PR.
Also, I actually find the 2-line version to be more readable somehow...
I don't feel strongly about it though, let me know if you do!

Reply:
No strong opinion. It was just a nit, maybe for the future. I'm fine with leaving like this